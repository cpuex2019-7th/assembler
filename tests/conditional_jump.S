    addi t0, zero, 1
    addi t1, zero, 1
    beq t0, t1, eq
    j neq
eq:
    addi t2, zero, 3
    j ret
neq:
    addi t2, zero, 4
    j ret
ret:
    addi a0, t2, 0 ; nop   
