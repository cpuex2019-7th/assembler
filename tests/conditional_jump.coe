memory_initialization_radix=16;
memory_initialization_vector=00100293,00100313,00628463,00c0006f,00300393,00c0006f,00400393,0040006f,00038513;
radix=16;
