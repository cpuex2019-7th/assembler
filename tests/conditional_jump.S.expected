addi t0, zero, 0x1
addi t1, zero, 0x1
beq t0, t1, 0x8 
jal zero, 0xc 
addi t2, zero, 0x3
jal zero, 0xc 
addi t2, zero, 0x4
jal zero, 0x4 
addi zero, zero, 0x1
